content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -0,0 +1,149 @@
+
+Next, click the :guilabel:`New Rule` button, and another line appears. In this second line, select
+:guilabel:`Current Skill Level` for the first drop-down, then select :guilabel:`Expert` for the
+third drop-down field.
+
+After the :guilabel:`New Rule` button is clicked, the word :guilabel:`"any"` in the sentence
+:guilabel:`Match any of the following rules:` changes from plain text into a drop-down menu. Click
+the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon after the word :guilabel:`any`, and select
+:guilabel:`all`.
+
+Finally, click the :guilabel:`Add` button.
+
+.. image:: skills_evolution/javascript.png
+   :align: center
+   :alt: The Custom Filter pop-up with the parameters set.
+
+Now, only employees that have an :guilabel:`Expert` level for the skill :guilabel:`Javascript`
+appear. In this example, only :guilabel:`Mark Demo` meets these criteria.
+
+.. image:: skills_evolution/results.png
+   :align: center
+   :alt: The employees with expert Javascript skills.
+
+Use case: Assess highest improvement
+====================================
+
+Another way to modify the :guilabel:`Appraisal Skills Report` is to identify the employee who has
+the highest amount of improved skills over a specific period of time. In this example, the employee
+with the most amount of improved skills for the third quarter is being determined.
+
+To view this information, first remove the default filter in the search bar. Next, click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
+:guilabel:`Improvement` beneath the :icon:`fa-filter` :guilabel:`Filters` column. Enabling this
+filter only presents skills that have improved.
+
+With the search bar still expanded, click :guilabel:`Add Custom Filter` at the bottom of the
+:icon:`fa-filter` :guilabel:`Filters` column, and an :guilabel:`Add Custom Filter` pop-up window
+appears.
+
+Select :guilabel:`Create Date` for the first drop-down field, then select :guilabel:`is between` for
+the second drop-down field. Once :guilabel:`is between` is selected, a second field appears after
+the last field. Using the calendar selector, set the first date to :guilabel:`07/01/2024` and the
+second date to :guilabel:`09/30/2024`. Once all the fields are properly formatted, click
+:guilabel:`Add`.
+
+.. image:: skills_evolution/custom-filter.png
+   :align: center
+   :alt: The Custom Filter pop-up with the parameters set.
+
+These filters present only the skills that have improved during the third quarter (between July 1st
+and September 30th 2024), organized by employee.
+
+The default list view may be difficult to view, depending on the number of employees and skills that
+are displayed. Click the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner to
+view the data in a pivot table. This presents a pivot table with the employees populating the rows,
+and the only visible column represents the total number of improved skills.
```

Comment:
in the two different 17.0 runbots I used to double-check this review, I'm not seeing a Pivot table option on this reporting page. Can you please look into this during your revisions? Thanks! 🙏